### PR TITLE
fixes a table climbing runtime

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
fixes a table climbing runtime

## Why It's Good For The Game
runtimes bad

## Testing
Climbed on a table

## Changelog
NPFC

### CODE REVIEW

File: code/game/objects/structures.dm [hunk 1]
```diff
@@ -82,42 +82,34 @@
...
+	user.Moved(old_loc, get_dir(old_loc, user.loc), FALSE)
 	if(get_turf(user) == get_turf(src))
-		if(HAS_TRAIT(climber, TRAIT_TABLE_LEAP))
+		if(HAS_TRAIT(user, TRAIT_TABLE_LEAP))
 			user.visible_message("<span class='warning'>[user] leaps up onto [src]!</span>")
 		else
 			user.visible_message("<span class='warning'>[user] climbs onto [src]!</span>")
 
-	climber = null
-	return TRUE
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/objects/structures.dm:117]: In the future someone can add that

File: code/game/objects/structures.dm [hunk 2]
```diff
@@ -82,42 +82,34 @@
...
-
 /obj/structure/proc/structure_shaken()
-
 	for(var/mob/living/M in get_turf(src))
```
[the remark this answers is not in the document]
Comment (GDNgit) [reply code/game/objects/structures.dm:114]: the other whitespace isn't as bad, this whitespace was a pain on my eyes

File: code/game/objects/structures.dm [hunk 3]
```diff
@@ -82,42 +82,34 @@
...
 	if(!can_touch(user) || !climbable)
-		climber = null
 		return FALSE
 
-	var/old_loc = usr.loc
-	user.loc = get_turf(src)
-	user.Moved(old_loc, get_dir(old_loc, usr.loc), FALSE)
+	var/old_loc = user.loc
+	user.forceMove(get_turf(src))
+	user.Moved(old_loc, get_dir(old_loc, user.loc), FALSE)
```
Comment (farie82) [inline code/game/objects/structures.dm:105]: is this one still needed due to the use of forceMove?

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please adress Farie's review, this is going stale

**Review by S34NW — CHANGES_REQUESTED** [review]
As above, address these reviews within a few days or this will be closed. PR is very stale.

File: code/game/objects/structures.dm (hunk 3 above)
Comment (GDNgit) [reply code/game/objects/structures.dm]: probably not

**Review by DGamerL — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**